src/Saas.Catalog/Saas.Catalog.Api/Models/CatalogTenant.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Saas.Domain.Models;
+
+using System;
+
+namespace Saas.Catalog.Api.Models
+{
+    public class CatalogTenant
```

Comment:
Do we need a catalogtenant vs just the domain model?

Reply:
I think so, they are the same right now, but having them separate would allow us to change the service without impacting other services in the future.  Also keeps the database structure from leaking over to other services.

Reply:
fair. Keeping it simple though, for mvp do we see that happening? Just wanted to keep it as lean as possible.  Either way, I'm good if you want to merge it. Keep it up!